Description: Link only Foundation library against -ldl and -lrt on Linux.
Origin: vendor
Forwarded: no
Author: Cristian Greco <cristian@debian.org>

--- a/build/config/Linux
+++ b/build/config/Linux
@@ -68,4 +68,4 @@
 #
 # System Specific Libraries
 #
-SYSLIBS  = -lpthread -ldl -lrt
+SYSLIBS  = -lpthread
--- a/Foundation/Makefile
+++ b/Foundation/Makefile
@@ -55,6 +55,10 @@
 	objects += SyslogChannel
 endif
 
+ifeq ($(findstring Linux, $(POCO_CONFIG)), Linux)
+	SYSLIBS += -ldl -lrt
+endif
+
 target         = PocoFoundation
 target_version = $(LIBVERSION)
 target_libs    =
